gadopt/utility.py

```diff
@@ -191,31 +191,6 @@ def cell_size(mesh):
         return CellDiameter(mesh)
 
 
-def cell_edge_integral_ratio(mesh, p):
```

Comment:
Is there a reason for deleting this? I think Steph has a branch working on limiters that may use this, but there's a chance I'm wrong.

Reply:
I moved it to the `equation` module because it was only used for the interior penalty method found in some specific terms. Might not be the best idea, but I felt like it did not belong in `utility`. By the way, `utility` is quite large now, and perhaps it should be broken up into multiple modules.